### ISSUE TITLE
Demands ipv6 forwarding even when ipv6 doesn't exist

### ISSUE THREAD

**Issue by bronson** [tailscale/tailscale#1999]
Tailscale demands that ipv6 forwarding be turned on, even though it knows that ipv6 isn't in use.

I have ipv4 forwarding turned on:

```
root:~# cat /proc/sys/net/ipv4/ip_forward
1
root:~# cat /proc/sys/net/ipv6/conf/all/forwarding 
0
```

And Tailscale recognizes that ipv6 doesn't exist.

```
root:~# tailscale netcheck
Report:
	* UDP: true
	* IPv4: yes, IP.IP.IP.IP:33498
	* IPv6: no
	* MappingVariesByDestIP: false
	...
```

Yet it says

![image](https://user-images.githubusercontent.com/1776/119874855-ea6b9880-beda-11eb-85c9-1f80f3635f24.png)

Sure enough, `echo 1 > /proc/sys/net/ipv6/conf/all/forwarding` fixes the issue.

It would be nice if Tailscale ignored ipv6 settings when ipv6 isn't in use.  (and it probably also makes sense to ignore ipv4 settings when ipv4 isn't in use!)

**Comment by DentonGentry** [comment]
Is it just the annoyance factor, that it is complaining about something that ultimately won't matter, or is there harm in having ipv6_forwarding enabled?

**Comment by bronson** [comment]
Agreed, I don't think there's any harm in enabling ipv6. It's just that, in my case, enabling it isn't so easy.

These hosts are installed from a stripped down Debian and are supposed to keep to the defaults (so they're all identical). Whoever enabled ipv4 forwarding probably just forgot ipv6. But this means that making ipv6 forwarding persistent requires modifying and redeploying the base image.

For now, because I'm only experimenting, I don't mind enabling ipv6 forwarding by hand. Besides, this self-imposed difficulty might be specific to this one ISP. Happy to close if it's not affecting anyone else.
